libs/ngx-charts-on-fhir/src/lib/fhir-mappers/medication-request/duration-medication-mapper.service.ts

```diff
@@ -207,12 +207,15 @@ export class DurationMedicationMapper implements Mapper<DurationMedication> {
     const layer = this.baseMapper.map(resource) as DataLayer<'bar', MedicationDataPoint[]>;
     layer.datasets = layer.datasets.map((dataset) => ({
       ...dataset,
-      label: dataset.label + ' *',
+      label: dataset.label,
```

Comment:
I think it will combine simple and duration medications into the same dataset if they have the same label. We don't want that. Did you test this with a patient who has both simple and duration observations of the same medication?

Reply:
I tested this and the duration bars do not show up for medications with mixed simple/duration observations if they have the same dataset label. We need to add something to the label to keep durations in a separate dataset. I suggest: 
```label: dataset.label + ' (Est. Duration)',```